.reuse/dep5

```diff
@@ -0,0 +1,10 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Source: https://github.com/RevolutionPi/revpi-pitest
+
+Files: doc/piTest.1
+Copyright: 2016-2023 KUNBUS GmbH, Heerweg 15C, 73770 Denkendorf, Germany
```

Comment:
Normally we don't use the full address in the copyright header.

Reply:
The copyright information is taken from the previously existing copyright notices. We can change this, but I would need to know what we put there instead.

My guess is just `KUNBUS GmbH`?